api/pkg/api/socket_server_handlers.go

```diff
@@ -18,6 +20,27 @@ type RoutesEventHandlers struct {
 	SocketHarOutChannel chan<- *tap.OutputChannelItem
 }
 
+func init() {
+	go updateAnalyzeStatus()
+}
+
+func updateAnalyzeStatus() {
```

Comment:
Move to a different package.
Otherwise it would be even more difficult to uncouple UP9 from Mizu.